CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 0.8.2 (2025-10-31)
+--------------------------
+Switch file mode to memory (#180)
```

Comment:
You're missing `Fix CI` commit

Reply:
🤦 I wrote it but didn't save the file! thanks! fixing now